app/models/user/MultiUser.scala

```diff
@@ -91,7 +93,8 @@ class MultiUserDAO @Inject()(sqlClient: SqlClient)(implicit ec: ExecutionContext
       _ <- run(q"""UPDATE webknossos.multiusers
                    SET
                      email = $email,
-                     isEmailVerified = false
+                     isEmailVerified = false,
+                     emailChangeDate = NOW()
```

Comment:
I think it would be helpful to add logging for this (possibly in the spot where updateEmail is called in UserService). Something along the lines of "MultiUser <id> changed their email from <old> to <new>.". This would allow us to verify claims of users whose emails were accidentally changed to email adresses they don’t control. Since our server logs are neither stored indefinitely nor unprotected, I’d say that this logging would be fair. What do you think?

Reply:
> Since our server logs are neither stored indefinitely nor unprotected, I’d say that this logging would be fair. What do you think?

I would say that's fine as it helps to improve security in edge cases like:
> This would allow us to verify claims of users whose emails were accidentally changed to email addresses they don’t control.

Which is also in favour for the users. As long as we do not abuse this info this  should be fine IMO. 
Double checking with @normanrz  :)

Reply:
cool, thanks :) could you include the multiUserId as well?

Reply:
okidoki done